ui/apps/platform/src/Containers/Collections/CollectionsTable.tsx

```diff
@@ -0,0 +1,131 @@
+                                        icon={SearchIcon}
+                                    >
+                                        <Text>Clear all filters and try again.</Text>
+                                        <Button variant="link" onClick={() => {}}>
+                                            Clear all filters
+                                        </Button>
+                                    </EmptyStateTemplate>
+                                </Bullseye>
+                            </Td>
+                        </Tr>
+                    )}
+                    {collections.map(({ id, name, description, inUse }, rowIndex) => {
+                        const actionItems = [
+                            {
+                                title: 'Edit collection',
+                                onClick: () => onEditCollection(id),
+                            },
+                            {
+                                title: 'Clone collection',
+                                onClick: () => onCloneCollection(id),
+                            },
+                            {
+                                isSeparator: true,
+                            },
+                            {
+                                title: inUse ? 'Cannot delete (in use)' : 'Delete collection',
+                                onClick: () => {},
+                                isDisabled: inUse,
+                            },
+                        ];
+
+                        return (
+                            <Tr key={id}>
+                                {hasWriteAccess && (
+                                    <Td
+                                        select={{
+                                            rowIndex,
+                                            onSelect,
+                                            isSelected: selected[rowIndex],
+                                        }}
+                                    />
+                                )}
+                                <Td dataLabel="Collection">
+                                    <Button
+                                        variant={ButtonVariant.link}
+                                        isInline
+                                        component={LinkShim}
+                                        href={`${collectionsPath}/${id}`}
+                                    >
+                                        {name}
+                                    </Button>
+                                </Td>
+                                <Td dataLabel="Description">
+                                    <Truncate content={description || '-'} tooltipPosition="top" />
+                                </Td>
+                                <Td dataLabel="In Use">{inUse ? 'Yes' : 'No'}</Td>
+                                {hasWriteAccess && <Td actions={{ items: actionItems }} />}
```

Comment:
Having this `{hasWriteAccess && ` check three times throughout here doesn't feel great, but seems more straightforward than abstracting the permission checks in some way.

Reply:
Yes, it is clear and findable.

This comment reminds me there might be another, for a corresponding `Th` element.
* `<Th aria-label="Row actions" />`
* `Td` might like to nave `className="pf-u-text-align-right"` prop so kabob aligns at the right, in case descriptions are short and page is wide.

Reply:
Good catch on the missing `<Th>`. It looks like the description column always fills the available space though, so the kebab is always right aligned. Is there a situation you've seen where the kebab does not right align?

Reply:
Oh, good. Aha, description does not have width. Maybe because I specified width for every column.